lib/components/FileAttachment/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useRef } from "react";
+
+import { saveAs } from "file-saver";
+
+import directUploadsApi from "apis/direct_uploads";
+import { DIRECT_UPLOAD_ENDPOINT } from "common/constants";
+import Button from "components/Common/Button";
+import useUppyUploader from "hooks/useUppyUploader";
+
+import Attachment from "./Attachment";
+import {
+  DEFAULT_UPPY_CONFIG,
+  DROP_DOWN_OPTIONS_KEYS,
+  UPPY_UPLOAD_CONFIG,
+} from "./constant";
+
+import Progress from "../Editor/MediaUploader/Progress";
+
+const FileAttachment = ({
+  endpoint = DIRECT_UPLOAD_ENDPOINT,
+  attachments = [],
+  onChange = _ => {},
+}) => {
+  const fileInputRef = useRef(null);
+  const uppyConfig = { ...DEFAULT_UPPY_CONFIG };
+
+  const { uppy, isUploading } = useUppyUploader({
+    endpoint,
+    uppyConfig,
+    uppyUploadConfig: { ...UPPY_UPLOAD_CONFIG },
+  });
+
+  const onUpload = async event => {
+    try {
+      for (const file of Array.from(event.target.files)) {
+        uppy.addFile({
+          name: file.name,
+          type: file.type,
+          data: file,
+        });
+      }
+      const { successful = [] } = await uppy.upload();
+
+      const uploadedFiles = successful.map(uploadedFile => ({
+        filename: uploadedFile.name,
+        signedId: uploadedFile.response.signed_id,
+        contentType: uploadedFile.type,
+        url: uploadedFile.response.blob_url,
```

Comment:
This isn't the neeto way of doing it. The snake case variables shouldn't be appearing inside code. 

Here is an example on how it can be fixed.
```jsx
({ name, signed_id: signedId }) => {}
```

Next time keep this in mind.